app/src/main/java/org/fossasia/openevent/general/ticket/TicketsFragment.kt

```diff
@@ -88,15 +87,24 @@ class TicketsFragment : Fragment() {
         rootView.register.setOnClickListener {
             val fragment = AttendeeFragment()
             val bundle = Bundle()
-            bundle.putLong("EVENT_ID", id)
-            bundle.putLong("TICKET_ID", ticketId.toLong())
+            bundle.putLong(EVENT_ID, id)
+            bundle.putSerializable(TICKET, ticket)
```

Comment:
It's confusing because you are not putting ticket here, naming variables correct;y is extremely importnt

Reply:
okay i will rename accordingly :)